languages/common-lisp/reference/README.md

```diff
@@ -51,12 +60,12 @@ document][csharp-example] as an example to base my work off of.
   - Sequences
 
 ### CLOS (Needs Some Work)
-  - Classes
+  - [Classes][clas]
```

Comment:
small nit - can we just use whole words? for the links? 'class', 'method' etc.

'meth' to me makes me think Crystal Meth (or Crystal Mehtod if I'm thinking musically) - neither seems appropriate here :)

Reply:
That's a very valid point. The world of Lisp is a wild one, but perhaps not that sort of wild :)

Reply:
Definitely!